Data/index.html

```diff
@@ -30,7 +30,7 @@
 </head>
 <body>
 <div class="head">
-<h1>cf-metadata-mailman by date</h1>
+<h1>Archive of all CF discussions from 2006 to 2019, including questions, standard names, and enhancements</h1>
```

Comment:
Hi @erget, @JonathanGregory 

I'm not sure about this. It might be fine, but the original text "cf-metadata-mailman by date" would change to. e.g. "cf-metadata-mailman by author" when you choose a different search criterion on the web page, at least as far as I recall trying out [Jonathan's web page](http://www.met.rdg.ac.uk/~jonathan/CF_metadata/cf-metadata-mailman/) - it 404s now, so I can't check!

Reply:
Ahh ok, I'll leave off for now - maybe it'd be best just to revert the commit, who knows what else we might break. I had thought that this was just the text for the header without semantics. Should I revert the commit?

Reply:
I've made a [new PR](https://github.com/cf-convention/mailing-list-archive/pull/2). Please see https://github.com/cf-convention/cf-convention.github.io/issues/222.